match_sorts_to_ref.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import argparse
+from dateutil.parser import *
```

Comment:
ha, i learned something! 🥇

Reply:
oh - the dateutil parser? it's pretty handy. I don't think I actually implemented it in this script tho....